universe/worlds/worlds.go

```diff
@@ -77,6 +79,79 @@ func (w *Worlds) GetWorlds() map[uuid.UUID]universe.World {
 	return worlds
 }
 
+func (w *Worlds) GetOptions(spaces map[uuid.UUID]universe.Space) ([]dto.ExploreOption, error) {
+	options := make([]dto.ExploreOption, 0, len(spaces))
+
+	for _, space := range spaces {
+		var description any
+
+		nameAttributeID := entry.NewAttributeID(universe.GetSystemPluginID(), universe.SpaceAttributeNameName)
+		nameValue, nameOk := space.GetSpaceAttributeValue(nameAttributeID)
+		if !nameOk {
+			return nil, errors.Errorf("could not get name value %q", nameValue)
+		}
+
+		if nameValue == nil {
+			return nil, errors.Errorf("spaceValue not found %q", nameValue)
+		}
+
+		descriptionAttributeID := entry.NewAttributeID(universe.GetSystemPluginID(), universe.SpaceAttributeNameDescription)
+		descriptionValue, _ := space.GetSpaceAttributeValue(descriptionAttributeID)
```

Comment:
bad practice to skip checks like here, because it's not guaranted that if future this method can't return broken value in case of `!ok`

Reply:
this check must be skipped for now, if i implement check then everything must have description
at the moment i don't think this is enforced

Reply:
spoke to Anton, this will be enforced so i re-added the check